ckanext/stadtzhtheme/plugin.py

```diff
@@ -746,8 +749,39 @@ def _set_resource_filename(self, resource):
             if upload.filename:
                 resource["filename"] = os.path.basename(upload.filename)
 
+    def _set_markdown_snippet_text(self, resource):
+        if not resource.get("markdown_snippet"):
+            package_name = (resource.get("package_id"),)
+            resource_id = (resource.get("id"),)
+            file_format = (resource.get("format"),)
+            download_url = (resource.get("url"),)
```

Comment:
We define these variables here, but only use them further down if the resource format is `csv` or `parquet` - can we move the definitions into that `elif` as well, or just skip definining the variables as we only use them once?

Reply:
Oh my, thank you for catching that! Setting those variables was intentional as I wanted to use them in the second part, too, but forgot to replace. Will change it accordingly, so you know what I mean 😅